rebar.config

```diff
@@ -16,8 +16,8 @@
 {profiles, [
     {local, []},
     {test, [
-        {deps, [{aesophia, {git, "https://github.com/aeternity/aesophia.git", {ref,"42cd47d"}}},
-                {aesophia_cli, {git, "git://github.com/aeternity/aesophia_cli", {tag, "v4.3.1"}}}]},
+        {deps, [{aesophia, {git, "https://github.com/aeternity/aesophia.git", {ref,"98a4049"}}},
```

Comment:
fe5f554 is the latest commit, as of 27 minutes ago

Reply:
Aim. Fire. Delay. Miss :)